utilities/vrt_helper.py

```
@@ -103,7 +95,7 @@ def shoot_page_ang_ignore_elements(
                 ).testRunResponse.status.name,
             )
 
-    def shoot_element(self, baseline_name: str, locator: Tuple[str, str]):
+    def shoot_element(self, baseline_name: str, locator: tuple[str, str]):
```

Ruff (0.8.2)

Missing return type annotation for public function `shoot_element`

Add return type annotation: `None`

(ANN201)